library/std/src/sys/pal/uefi/tests.rs

```diff
@@ -23,19 +25,60 @@ fn align() {
 }
 
 #[test]
-fn epoch() {
-    let t = r_efi::system::Time {
-        year: 1970,
+fn systemtime_start() {
```

Comment:
Do these tests get run currently? How can I run them locally?

Reply:
Well, I am not sure. I have not yet found any way at least. I run the tests in a seperate rust crate and just copy them here for now.

Reply:
It would be good to add a comment to that effect in this file, otherwise it's not obvious that the tests won't catch errors if you don't manually run them in a separate project.